components/calendar/src/error.rs

```diff
@@ -0,0 +1,37 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use displaydoc::Display;
+
+#[cfg(feature = "std")]
+impl std::error::Error for DateTimeError {}
+
+/// A list of possible error outcomes for working with various inputs to DateTime inputs
+/// and operations.
+#[derive(Display, Debug)]
+pub enum DateTimeError {
+    /// An input could not be parsed.
+    #[displaydoc("{0}")]
+    Parse(core::num::ParseIntError),
+    /// An input overflowed its range.
+    #[allow(missing_docs)] // TODO(#686) - Add missing docs.
```

Comment:
Nit: Here and below: why do you need the `#[allow(missing_docs)]` ?

Reply:
It's a copy paste, I'll clean that up later